server/jvm/{{appName}}-config/build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 dependencies {
-    compileOnly("global.genesis:genesis-dictionary")
-    compileOnly("global.genesis:genesis-process")
-    compileOnly("global.genesis:genesis-pal-execution")
-    compileOnly(project(path = ":{{appName}}-dictionary-cache", configuration = "codeGen"))
+    compileOnly(genesis("genesis-dictionary"))
+    compileOnly(genesis("genesis-process"))
+    compileOnly(genesis("genesis-pal-execution"))
```

Comment:
This `genesis` function has recently changed, it now works like so.
`genesis("dictionary")` will create `"global.genesis:genesis-dictionary"`